README.md

```diff
@@ -55,10 +55,22 @@ USAGE
   $ sync-db
 
 OPTIONS
-  -f, --force                 Force synchronization
```

Comment:
Has this option been removed?

Reply:
Missed the PR with this change. Nevermind.

Reply:
No added it back.

Reply:
😄